### PR TITLE
Feature seg json selector

### PR DESCRIPTION
**Pull request by deepakri201**
@fedorov I have made the following changes: 
- I modified the segmentation json terminology file to reflect the correct DICOM CodeMeanings/CodingSchemeDesignators 
- I implemented a file selector button for the user to choose a terminology file of their choice. If a file is not selected, the default terminology (for prostate cancer) is used

### CODE REVIEW

File: mpReview.py [hunk 1]
```diff
@@ -325,6 +325,26 @@ def setupDatabaseSelectionView(self):
     self.databaseGroupBox.setLayout(databaseGroupBoxLayout)
     self.databaseSelectionWidgetLayout.addWidget(self.databaseGroupBox, 3, 0, 1, 3)
     
+    ###############################
+    ### Select terminology file ### 
+    ###############################
+    
+    self.terminologyGroupBox = qt.QGroupBox("Terminology")
+    terminologyGroupBoxLayout = qt.QFormLayout()
+    self.selectTerminologyFileButton = qt.QPushButton("Select Segmentation JSON Terminology file")
```
Comment (fedorov) [inline mpReview.py]: Maybe it's better to use `ctkPathLineEdit` instead?

File: mpReview.py [hunk 2]
```diff
@@ -325,6 +325,26 @@ def setupDatabaseSelectionView(self):
...
+    self.selectTerminologyFileButton.setEnabled(True)
+    terminologyGroupBoxLayout.addRow(self.selectTerminologyFileButton)
+    
+    self.terminologyGroupBox.setLayout(terminologyGroupBoxLayout)
+    self.databaseSelectionWidgetLayout.addWidget(self.terminologyGroupBox, 4, 0, 1, 3)
+    
+  def selectTerminologyFile(self):
+    """ open a QFileDialog box where the user can optionally choose a segmentation JSON terminology file """
+    segJsonPrompt = qt.QFileDialog()
+    self.terminologyFile = segJsonPrompt.getOpenFileName(None,"Select File", "", "All Files (*);;Python Files (*.json)")
```
Comment (fedorov) [inline mpReview.py]: Should "Python files" be "JSON files"?

File: mpReview.py [hunk 3]
```diff
@@ -896,9 +917,13 @@ def setupConnections(self):
 
   def checkAndSetLUT(self):
```
Comment (fedorov) [inline mpReview.py:945]: For the future - it would be good to validate that the selected JSON file in fact contains the expected content!

**Review by fedorov — CHANGES_REQUESTED** [review]

File: mpReview.py (hunk 1 above)
Comment (deepakri201) [reply mpReview.py]: Good idea, I changed the code to use ctkPathLineEdit instead of QPushButton+QFileDialog

File: mpReview.py (hunk 2 above)
Comment (deepakri201) [reply mpReview.py]: This is now changed as I am using ctkPathLineEdit -- I did the following to make sure only json files are displayed when choosing the terminology file: `self.terminologyFilePathLineEdit.nameFilters = ['*.json']`

File: mpReview.py (hunk 3 above)
Comment (deepakri201) [reply mpReview.py:945]: Good point, I included some basic checks to see if certain fields exist in the terminology json file. If any fail, a warning is printed, and the default (prostate) one is used instead.

**Comment by fedorov** [comment]
Feel free to merge!

**Review by fedorov — APPROVED** [review]

**Pull request merged**